.github/workflows/ci.yml

```diff
@@ -1,6 +1,12 @@
 ---
 name: Test
-on: [push]
+on:
+  push:
+    branches:
+      - master
```

Comment:
Hum, that's the reason I changed it, I was annoyed not to have CI when pushing in other branches. 

Maybe the best would be `on: [push, pull_request]`?

Reply:
- [ ] on: [push, pull_request]

Reply:
fixed: 5f0edc7